Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -102,3 +102,38 @@ the pressure drop of the heating coil to zero, and
 to lump the pressure drop of the heating coil into the pressure drop model
 of the cooling coil.
 This often reduces the size of the system of nonlinear equations.
+
+Working process
+---------------
+
+Followings are the working process of contributing.
+
+1. create issue to describe the potential problem or what change should be done
+   to the library. This will generate a issue number (``##``). A good
+   practice is that each issue should address only one problem.
+2. create development branch,
+
+  - if the issue is not part of a large development, the development branch
+    should be named as ``issue(##)_(change)`` and should be based on the ``master``
+    branch.
+  - if the issue is part of a large development and the development has a
+    staging branch like ``issue(#)_(development)``, the development branch should
+    be named as ``issue(#)_(##)_(change)`` and should be based on the staging
+    branch.
+
+3. address the issue,
+
+  - fork ``Buildings`` library repository to your account, or pull the development
+    branch to your own ``Buildings`` library repository,
+  - make the change from the development branch,
+  - submit a pull request to merge the changes in the development branch of
+    your own ``Buildings`` library repository to the same development branch of
+    ``Buildings`` library repository,
+  - merge the pull request.
+
+4. close the issue in ``Buildings`` library repository,
+
+  - submit another pull request to merge the development branch to
+    ``master`` branch,
+  - merge the pull request,
```

Comment:
Here it is for the project manager who will eventually merge the staging branch: do we need to describe that process?